ols-vms.conf

```diff
@@ -0,0 +1,11 @@
+vm.architecture = amd64
+vm.release = focal
+
+[serial-vault]
+# PPAs for python-codetree and charm-tools
+apt.sources = ppa:mojo-maintainers/ppa, ppa:juju/stable
+vm.packages = @dependencies.txt, @dependencies-devel.txt, build-essential
+
+# sideload snaps
```

Comment:
Forgot about this one, no need to include the lines below if no snaps are required.

Reply:
I will use golang 1.13 from the snap because we don't have this version on xenial